<!doctype html>
<html>
  <head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="chrome=1">
    <title>Mreppell.GitHub.io by mreppell</title>
    <link rel="stylesheet" href="stylesheets/styles.css">
    <link rel="stylesheet" href="stylesheets/pygment_trac.css">
    <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.7.1/jquery.min.js"></script>
    <script src="javascripts/respond.js"></script>
    <!--[if lt IE 9]>
      <script src="//html5shiv.googlecode.com/svn/trunk/html5.js"></script>
    <![endif]-->
    <!--[if lt IE 8]>
    <link rel="stylesheet" href="stylesheets/ie.css">
    <![endif]-->
    <meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no">

  </head>
  <body>
      <div id="header">
        <nav>
          <li class="fork"><a href="https://github.com/mreppell">View On GitHub</a></li>
        </nav>
      </div><!-- end header -->

    <div class="wrapper">

      <section>
        <div id="title">
          <h1>Mreppell.GitHub.io</h1>
          <p></p>
          <hr>
          <span class="credits left">Project maintained by <a href="https://github.com/mreppell">mreppell</a></span>
          <span class="credits right">Hosted on GitHub Pages &mdash; Theme by <a href="https://twitter.com/michigangraham">mattgraham</a></span>
        </div>

        <p><br></p>

<p><img src="cloud-gate.jpg" alt="Unable to display image"></p>

<p><br></p>

<p>For a complete listing please see <a href="https://scholar.google.com/citations?user=GIZ2jKwAAAAJ&amp;hl=en&amp;citsig=AMstHGRJSV7QIE4egZazcqLFcxyU9Z7inA">Google Scholar</a></p>
<p><br></p>
<div id="recent-publications" class="section level2">
<h2>Recent Publications</h2>
<p><br></p>

<p><br></p>

<div id="reppell-m-koch-e-peter-bm-novembre-j.-2014-surfing-waves-of-data-in-san-diego-sophisticated-analyses-provide-a-broad-view-of-human-genetic-diversity.-genome-biology.-15562" class="section level3">
<h3>Reppell M, Koch E, Peter BM, Novembre J. (2014) Surfing waves of data in San Diego: sophisticated analyses provide a broad view of human genetic diversity. <em>Genome Biology</em>. 15:562</h3>
<p>A report on the 64th annual American Society of Human Genetics meeting held in San Diego, USA, 18-22 October, 2014.</p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/25651070">PMID: 25651070</a></p>
<p><br></p>
</div>
<div id="zawistowski-mreppell-m-wegmann-d-st-jean-pl-ehm-mg-nelson-mr-novembre-j-zollner.-2014-analysis-of-rare-variant-population-structure-in-europeans-explains-differential-stratification-of-gene-based-tests.-european-journal-of-human-genetics.-22-1137-1144" class="section level3">
<h3>Zawistowski M*,Reppell M*, Wegmann D, St Jean PL, Ehm MG, Nelson MR, Novembre J, Zollner. (2014) Analysis of rare variant population structure in Europeans explains differential stratification of gene-based tests. <em>European Journal of Human Genetics</em>. 22, 1137-1144</h5>
<p>There is substantial interest in the role of rare genetic variants in the etiology of complex human diseases. Several gene-based tests have been developed to simultaneously analyze multiple rare variants for association with phenotypic traits. The tests can largely be partitioned into two classes - ‘burden’ tests and ‘joint’ tests - based on how they accumulate evidence of association across sites. We used the empirical joint site frequency spectra of rare, nonsynonymous variation from a large multi-population sequencing study to explore the effect of realistic rare variant population structure on gene-based tests. We observed an important difference between the two test classes: their susceptibility to population stratification. Focusing on European samples, we found that joint tests, which allow variants to have opposite directions of effect, consistently showed higher levels of P-value inflation than burden tests. We determined that the differential stratification was caused by two specific patterns in the interpopulation distribution of rare variants, each correlating with inflation in one of the test classes. The pattern that inflates joint tests is more prevalent in real data, explaining the higher levels of inflation in these tests. Furthermore, we show that the different sources of inflation between tests lead to heterogeneous responses to genomic control correction and the number of variants analyzed. Our results indicate that care must be taken when interpreting joint and burden analyses of the same set of rare variants, in particular, to avoid mistaking inflated P-values in joint tests for stronger signals of true associations.</p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/24398795">PMID: 24398795</a></p>
<p><br></p>
</div>
<div id="reppell-m-boehnke-m-zollner-s.-2013-the-impact-of-accelerating-faster-than-exponential-population-growth-on-genetic-variation.-genetics.-196-819-828." class="section level3">
<h3>Reppell M, Boehnke M, Zollner S. (2013) The impact of accelerating, faster than exponential population growth on genetic variation. <em>Genetics</em>. 196, 819-828.</h5>
<p>Current human sequencing projects observe an abundance of extremely rare genetic variation, suggesting recent acceleration of population growth. To better understand the impact of such accelerating growth on the quantity and nature of genetic variation, we present a new class of models capable of incorporating faster than exponential growth in a coalescent framework. Our work shows that such accelerated growth affects only the population size in the recent past and thus large samples are required to detect the models’ effects on patterns of variation. When we compare models with fixed initial growth rate, models with accelerating growth achieve very large current population sizes and large samples from these populations contain more variation than samples from populations with constant growth. This increase is driven almost entirely by an increase in singleton variation. Moreover, linkage disequilibrium decays faster in populations with accelerating growth. When we instead condition on current population size, models with accelerating growth result in less overall variation and slower linkage disequilibrium decay compared to models with exponential growth. We also find that pairwise linkage disequilibrium of very rare variants contains information about growth rates in the recent past. Finally, we demonstrate that models of accelerating growth may substantially change estimates of present-day effective population sizes and growth times.</p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/24381333">PMID: 24381333</a></p>
    
	</div>
    <!--[if !IE]><script>fixScale(document);</script><![endif]-->
    
  </body>
</html>
